*** tests/tests_public/A-01-02-M-ok.out	2024-10-09 09:12:10.000000000 +0100
--- tests/tests_public/A-01-02-M-ok.outhyp	2024-10-25 03:56:17.021084260 +0100
***************
*** 43,59 ****
  10 - Consultas
  0 - Sair
  Escolha uma opção: 3
- Ficheiro sem nome. Guardar ficheiro como: app01.dat
- Menu Principal
- 1 - Criar ficheiro
- 2 - Abrir ficheiro
- 3 - Guardar ficheiro
- 4 - Avançar estação do ano
- 5 - Ver satisfação global
- 6 - Gestão de animais
- 7 - Gestão de funcionários
- 8 - Gestão de habitats
- 9 - Gestão de vacinas
- 10 - Consultas
- 0 - Sair
- Escolha uma opção: 0
--- 43,45 ----
