*** tests/tests_final/A-03-03-M-ok.out	2024-10-20 12:31:15.000000000 +0100
--- tests/tests_final/A-03-03-M-ok.outhyp	2024-10-25 16:08:41.999408727 +0100
***************
*** 34,40 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Nome do habitat: Área do habitat: Registar habitat: Operação inválida: O habitat 'z' já existe.
  Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
--- 34,40 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Nome do habitat: Área do habitat: Registar habitat: Operação inválida: O habitat 'idHabitat' já existe.
  Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
