*** tests/tests_final/A-03-01-M-ok.out	2024-10-20 12:31:15.000000000 +0100
--- tests/tests_final/A-03-01-M-ok.outhyp	2024-10-25 03:51:24.504434380 +0100
***************
*** 18,24 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Nome do habitat: Área do habitat: Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
--- 18,24 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
***************
*** 26,50 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
- Escolha uma opção: HABITAT|1|primeiro|1|0
- Gestão de Habitats
- 1 - Visualizar todos os habitats
- 2 - Registar habitat
- 3 - Alterar área de habitat
- 4 - Alterar influência de habitat sobre espécie
- 5 - Plantar árvore
- 6 - Mostrar árvores de habitat
- 0 - Sair
- Escolha uma opção: Menu Principal
- 1 - Criar ficheiro
- 2 - Abrir ficheiro
- 3 - Guardar ficheiro
- 4 - Avançar estação do ano
- 5 - Ver satisfação global
- 6 - Gestão de animais
- 7 - Gestão de funcionários
- 8 - Gestão de habitats
- 9 - Gestão de vacinas
- 10 - Consultas
- 0 - Sair
  Escolha uma opção: 
\ No newline at end of file
--- 26,29 ----
