*** tests/tests_final/A-21-02-M-ok.out	2024-10-20 12:31:15.000000000 +0100
--- tests/tests_final/A-21-02-M-ok.outhyp	2024-10-25 23:08:43.104324674 +0100
***************
*** 16,22 ****
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador da vacina: Identficador do veterinário a ministrar a vacina: Identificador único do animal: Vacinar animal: Operação inválida: O veterinário 'Trt1' não existe.
  Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
--- 16,22 ----
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador da vacina: Identficador do veterinário a ministrar a vacina: Identificador único do animal: Vacinar animal: Operação inválida: O veterinário 'Trt1' não pode ministrar vacinas à espécie 'Trt1'
  Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
